src/pages/UserMyVisits.jsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
-import { Layout } from '../components';
+import { Layout, Sidebar } from '../components';
+import { Box, Toolbar, Typography } from '@mui/material';
 
 export const UserMyVisits = () => {
-  return <Layout>My Visits</Layout>;
+  return (
+    <>
+      <Layout />
+      <Sidebar />
+      <Box component="main" sx={{ flexGrow: 1, p: 3 }}>
+        <Toolbar />
+        <Typography paragraph>My visits</Typography>
+      </Box>
```

Comment:
You repeat the same code in some places. I recommend to move this to Layout component.